:100644 100644 1efc801... 0000000... M	documentation/manual/en/module_specs/Zend_Tool_Framework-WritingProviders.xml

diff --git a/documentation/manual/en/module_specs/Zend_Tool_Framework-WritingProviders.xml b/documentation/manual/en/module_specs/Zend_Tool_Framework-WritingProviders.xml
index 1efc801..088463b 100644
--- a/documentation/manual/en/module_specs/Zend_Tool_Framework-WritingProviders.xml
+++ b/documentation/manual/en/module_specs/Zend_Tool_Framework-WritingProviders.xml
@@ -228,9 +228,9 @@ class My_Component_HelloProvider
         $nameResponse = $this->_registry
                              ->getClient()
                              ->promptInteractiveInput("Whats your name?");
-        $name = $name->getContent();
+        $name = $nameResponse->getContent();
 
-        echo 'Hello' . $name . ', from my provider!';
+        echo 'Hello ' . $name . ', from my provider!';
     }
 }
 ]]></programlisting>
